### ISSUE TITLE
Custom configuration path does not work

### ISSUE THREAD

**Issue by rernst** [SciLifeLab/genologics#201]
In https://github.com/SciLifeLab/genologics/pull/146 a flexible way declaring the configuration file location was implemented. Unfortunately, I can't get it to work.

This is how my (very simple) test script looks like:
```python
from genologics import config
```

When I run this it immediately starts complaining about the missing config files. 
```
/venv/lib/python2.7/site-packages/genologics/config.py:52: UserWarning: Please make sure you've created or indicated your own Genologics configuration file (i.e: ~/.genologicsrc) as stated in README.md
  warnings.warn("Please make sure you've created or indicated your own Genologics configuration file (i.e: ~/.genologicsrc) as stated in README.md")
```

I am using the latest (pip) version of the package. Any idea how to get this to work?

**Comment by Galithil** [comment]
Okay, can you tell me where is your configuration file located, so I can try to replicate the issue ?

**Comment by rernst** [comment]
It is located in the same directory as my python script. But this should
not matter. Without having one in one of the hardcoded places the code will
always crash.

Op vr 4 aug. 2017 11:17 schreef Denis Moreno <notifications@github.com>:

**Comment by Galithil** [comment]
Yes, you can only use the api if you provide a valid configuration to connect to a LIMS instance. We allow this file to be placed in several predetermined locations. 

I'm not quite sure what the issue is with this, could you explain your issue further ?

**Comment by rernst** [comment]
Sure! According to the docstring in config.py there is an alternative way
of supplying configuration files, which was implemented in the earlier
mentioned pull request. I can't get this alternative way to work so I was
wondering how I should implement it. The problem is that as soon as I try
to load config.py it will try to look for the configuration files and thus
the alternative method does not work.

This is the docstring:
```python
'''
Usage:

from genologics.config import BASEURI, USERNAME, PASSWORD

Alternate Usage:

from genologics import config

BASEURI, USERNAME, PASSWORD, VERSION, MAIN_LOG =
config.load_config(specified_config = <path to config file>)

'''
```

Op vr 4 aug. 2017 16:11 schreef Denis Moreno <notifications@github.com>:

**Comment by Galithil** [comment]
Interesting, as you noticed, that definitely won't work. 
We do _require_ a configuration file in one of the hardcoded locations. 
If you want to overwrite that one and provide your own, you need to do it on top of the hardcoded one. I'm guessing that is how the Bigelow lab handles this. 

I do not have an issue with allowing people to provide their own conf without using the default hardcoded ones, so if you're interested, you're welcome to put in a PR for that.
